apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1538,9 +1537,7 @@
   };
 
   MockNavigatormozMobileMessage.retrieveMMS = function(id) {
-    var request = {
-      error: null
-    };
+    var deferred = Utils.Promise.defer();
```

Comment:
Sorry my bad :/ , added in the latest commit and you can check if it works for you @azasypkin